crates/aquascope/src/interpreter/step.rs

```diff
@@ -120,7 +121,7 @@ impl<'tcx> MovedPlaces<'tcx> {
   }
 
   pub fn add_place(&mut self, frame: usize, place: Place<'tcx>) {
-    self.0.get_mut(frame).unwrap().insert(place);
```

Comment:
i don't think i fully understand how frames worked before vs how they work now. `Local`s no longer contain an explicit frame reference, i commented this out to avoid panics under the assumption that inserting the place would create incorrect visualizations but not harm whatever local state you have.